README.md

```diff
@@ -76,8 +76,9 @@ Import { Chat } from 'botframework-webchat';
 
 const YourApp = () => {
     <div>
+        <link href="https://unpkg.com/botframework-webchat/botchat.css" rel="stylesheet" />
```

Comment:
in this example I assume folks have `npm install`'ed WebChat, so this doesn't apply.